task/buildah-oci-ta/0.4/buildah-oci-ta.yaml

```diff
@@ -39,6 +39,11 @@ spec:
         to the image
       type: array
       default: []
+    - name: ANNOTATIONS_FROM_PATH
+      description: Path to a file with additional key=value annotations that
+        should be applied to the image
+      type: string
+      default: ""
```

Comment:
Consider renaming to `ANNOTATIONS_FILE` for symmetry with `BUILD_ARGS` -> `BUILD_ARGS_FILE`